src/EnergyPlus/ReportCoilSelection.cc

```diff
@@ -979,6 +909,61 @@ int ReportCoilSelection::getIndexForOrCreateDataObjFromCoilName(EnergyPlusData &
     return index;
 }
 
+void ReportCoilSelection::associateZoneCoilWithParent(EnergyPlusData &state, std::unique_ptr<CoilSelectionData> &c)
+{
+    c->coilLocation = "Unknown";
+    c->typeHVACname = "Unknown";
+    c->userNameforHVACsystem = "Unknown";
+    // now search equipment
+    auto const &zoneEquipList = state.dataZoneEquip->ZoneEquipList(c->zoneEqNum);
+    bool keepLooking = true;
+    for (int equipLoop = 1; equipLoop <= zoneEquipList.NumOfEquipTypes; ++equipLoop) {
+        if (zoneEquipList.EquipName(equipLoop) == c->coilName_) {
+            c->typeHVACname = zoneEquipList.EquipTypeName(equipLoop);
+            c->userNameforHVACsystem = zoneEquipList.EquipName(equipLoop);
+            c->coilLocation = "Zone Equipment";
+            keepLooking = false;
+            break;
+        }
+        if (keepLooking) {
+            if (zoneEquipList.EquipData(equipLoop).Name == c->coilName_) {
+                c->typeHVACname = zoneEquipList.EquipTypeName(equipLoop);
+                c->userNameforHVACsystem = zoneEquipList.EquipName(equipLoop);
+                c->coilLocation = "Zone Equipment";
+                keepLooking = false;
+                break;
+            }
```

Comment:
Likewise dead code here where a coil object cannot be the upper level Name in EquipData.

Reply:
So, you'll remove these dead blocks?